.scss-lint.yml

```diff
@@ -2,7 +2,7 @@
 
 scss_files: "assets/sass/**/*.scss"
 plugin_directories: ['.scss-linters']
-exclude: ["assets/sass/widgets/*.scss", "assets/sass/common/**/*.scss"]
+exclude: ["assets/sass/widgets/*.scss", "assets/sass/common/**/*.scss", "assets/sass/_container.scss", "assets/sass/okta-sign-in.scss"]
```

Comment:
We shouldn't ignore the whole file - just the line itself, i.e. http://stackoverflow.com/questions/32833542/having-scss-lint-ignore-a-particular-line